log4j2-ecs-layout/src/test/java/co/elastic/logging/log4j2/EcsLayoutWithExceptionPatternTest.java

```diff
@@ -0,0 +1,33 @@
+package co.elastic.logging.log4j2;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EcsLayoutWithExceptionPatternTest extends Log4j2EcsLayoutTest {
+    @Override
+    protected EcsLayout.Builder configureLayout(LoggerContext context) {
+        return super.configureLayout(context)
+                .setExceptionPattern("%cEx");
+    }
+
+    @Test
+    void testLogException() throws Exception {
+        error("test", new RuntimeException("test"));
+        JsonNode log = getLastLogLine();
+        assertThat(log.get("log.level").textValue()).isIn("ERROR", "SEVERE");
+        assertThat(log.get("error.message").textValue()).isEqualTo("test");
+        assertThat(log.get("error.type").textValue()).isEqualTo(RuntimeException.class.getName());
+        assertThat(log.get("error.stack_trace").textValue()).isEqualTo("java.lang.RuntimeException: test\nSTACK_TRACE!");
+    }
+
+    @Test
+    void testLogExceptionNullMessage() throws Exception {
+        error("test", new RuntimeException());
+        JsonNode log = getLastLogLine();;
+        assertThat(log.get("error.type").textValue()).isEqualTo(RuntimeException.class.getName());
+        assertThat(log.get("error.message")).isNull();
```

Comment:
```suggestion
        assertThat(log.get("error.message")).isNull();
        assertThat(log.get("error.stack_trace").textValue()).isEqualTo("java.lang.RuntimeException\nSTACK_TRACE!");
```
(didn't actually test that, but based on code - we need to test this `else` branch as well)